src/tsconfig.json

```diff
@@ -4,8 +4,9 @@
     "outDir": "../out",
     "allowJs": false,
     "sourceMap": true,
-    "target": "ES2016",
-    "strict": true
+    "target": "esnext",
+    "strict": true,
+    "lib": ["esnext", "esnext.string"]
```

Comment:
Since this is inheriting from `std/portable.json`, wdyt of making this change there so all portable AS projects can use `replaceAll`, not just the compiler?